### PR TITLE
[1LP][RFR] 511 refactor for migration tests

### PR DESCRIPTION
**Pull request by Yadnyawalkya**
{{pytest: cfme/tests/v2v/test_v2v_migrations.py --use-provider rhv43 --use-provider vsphere65-nested --provider-limit 2 -vvvv -k 'test_dual_disk_vm_migration'}}

- [x] Removing old model pages and fixtures since refactor is done
- [x] Replacing unnecessary functions with global fixture
- [x] Changing navigations for 511 (we have migration in main navigation now)
- [x] Fixing host authentication failure for 511

This fixes will rollout 511 jenkin job for v2v

### CODE REVIEW

**Comment by Yadnyawalkya** [comment]
@sshveta @mshriver  511 running now. @digitronik  can you please proceed with review.

File: cfme/fixtures/v2v_fixtures.py [hunk 1]
```diff
@@ -83,11 +85,21 @@ def __host_credentials(appliance, transformation_method, v2v_providers): # noqa
...
-                if not host_data:
+                try:
+                    host_data, = [
+                        data for data in v2v_provider.data["hosts"] if data["name"] == host.name]
+                except ValueError:
                     pytest.skip("No host data")
-                host.update_credentials_rest(credentials=host_data[0]['credentials'])
+                # TODO: Add host authentication via REST (BZ1718209)
+                # host.update_credentials_rest(credentials=host_data[0]['credentials'])
+                host_obj = appliance.collections.hosts.instantiate(
```
Comment (digitronik) [inline cfme/fixtures/v2v_fixtures.py]: @Yadnyawalkya I would use bz blocker
```
if not BZ(1718209).blocks:
    host.update_credentials_rest(credentials=host_data[0]['credentials'])
else:
   # with UI
```

Comment (Yadnyawalkya) [reply cfme/fixtures/v2v_fixtures.py]: Make sense, updated. :+1:

**Comment by Yadnyawalkya** [comment]
@izapolsk PRT for 5.10 is false positive but 5.11 PRT results are genuine and working right now. You can check.

**Comment by Yadnyawalkya** [comment]
@izapolsk  @mshriver this time everything is working perfectly fine on PRT, I fixed last KeyErrors, PR is ready.

**Comment by Yadnyawalkya** [comment]
Thanks @izapolsk  @digitronik .

**Pull request merged**